charts/tfy-gpu-operator/templates/gke-cos-gpu-driver.yaml

```
@@ -76,7 +76,7 @@ spec:
           hostPath:
             path: /etc/nvidia
       initContainers:
-        - image: "cos-nvidia-installer:fixed"
+        - image: "{{ .Values.gcpGkeStandardDriver.nvidiaDriverInstaller.cos.image.repository }}:{{ .Values.gcpGkeStandardDriver.nvidiaDriverInstaller.cos.tag }}"
```

These need to be changed now
I would recommend adding

```
{{- $driverDefault := (index .Values "gcp-gke-standard-driver" "default") }}
{{- $driverLatest := (index .Values "gcp-gke-standard-driver" "latest") }}
```

at the top

```suggestion
        - image: "{{ $driverDefault.cosDriverInstaller.image.repository }}:{{ $driverDefault.cosDriverInstaller.image.tag }}"
```